src/Compiler/Checking/NameResolution.fs

```diff
@@ -4063,7 +4063,15 @@ let private ResolveExprDotLongIdent (ncenv: NameResolver) m ad nenv ty (id: Iden
     | _ ->
         ForceRaise adhocDotSearchAccessible
 
-let ComputeItemRange wholem (lid: Ident list) rest =
+let ComputeItemRange (item: Item) wholem (lid: Ident list) rest =
+    match item, lid with
+    | Item.MethodGroup _, methodIdents -> 
```

Comment:
@edgarfgp Do you think checking that the item is a method group needed here? Could it also improve things for other item kinds?

Reply:
Yes we can remove this and more item kinds, but I will have more work to do 🙃